Libraries/Azure.psm1

```diff
@@ -228,8 +228,8 @@ Function Assert-ResourceLimitationForDeployment($RGXMLData, [ref]$TargetLocation
 
 Function Move-OsVHDToStorageAccount($OriginalOsVHD, $TargetStorageAccount) {
 	$sourceStorageAccount = $OriginalOsVHD.Replace("https://", "").Replace("http://", "").Split(".")[0]
-	$sourceContainer = $OriginalOsVHD.Split("/")[$OriginalOsVHD.Split("/").Count - 2]
-	$vhdName = $OriginalOsVHD.Split("?")[0].split('/')[-1]
+	$sourceContainer = $OriginalOsVHD.Split("/")[3]
```

Comment:
Make sure it compatible with both new and old formats.

Reply:
Yeah, I have tested for both new and old formats.

Reply:
ok. Add supported formats as comments above this line. So that people knows what format exactly supported in future.